docs/application/web/guides/device/system-information.md

```diff
@@ -282,6 +282,10 @@ The system properties are defined as subtypes of the `SystemInfoProperty` interf
 | `SystemInfoPeripheral` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoPeripheral), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoPeripheral), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoPeripheral) applications) | `PERIPHERAL`         | Provides information about the video output status. |
 | `SystemInfoMemory` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoMemory), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoMemory), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoMemory) applications) | `MEMORY`             | Provides information about the memory state of the device. |
 | `SystemInfoADS` (in [mobile](../../api/latest/device_api/mobile/tizen/systeminfo.html#SystemInfoADS), [wearable](../../api/latest/device_api/wearable/tizen/systeminfo.html#SystemInfoADS), and [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoADS) applications) | `ADS`                | Provides information about the advertisement service. |
+| `SystemInfoVideoSource` (in [TV](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSource) applications) | `VIDEOSOURCE`                | Provides information about connected and disconnected devices. |
```

Comment:
If only one profile is supported, you don't need to use "(in ~~ application)" form.
Applying link directly as like [SystemInfoVideoSource](../../api/latest/device_api/tv/tizen/systeminfo.html#SystemInfoVideoSource) is ok.